### PR TITLE
OSSMDOC-404 Update control plane profiles documentation

### PR DESCRIPTION
**Pull request by rh-tokeefe**
[OSSMDOC-404](https://issues.redhat.com/browse/OSSMDOC-404) Update control plane profiles documentation

- Aligned team: Dev Tools
- For branches: 4.6+ 
- Jira: https://issues.redhat.com/browse/OSSMDOC-404
- Direct link to doc preview: https://deploy-preview-36857--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/ossm-profiles-users?utm_source=github&utm_campaign=bot_dp#ossm-create-configmap_ossm-profiles-users
- SME review: @luksa
- QE review: @yxun
- Peer review: @rolfedh
- Cherry pick to 4.6, 4.7, 4.8, 4.9, 4.10

### CODE REVIEW

**Review by yxun — APPROVED** [review]

**Review by luksa — APPROVED** [review]

File: modules/ossm-control-plane-profiles.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ When you configure control plane profiles, which follow the same syntax as the `
 [id="ossm-create-configmap_{context}"]
 == Creating the ConfigMap
 
-To add custom profiles, you must first create a ConfigMap named `smcp-templates` in the `openshift-operators` project and then mount the ConfigMap in the Operator container at: `/usr/local/share/istio-operator/templates`.
+To add custom profiles, you must create a ConfigMap named `smcp-templates` in the `openshift-operators` project. The Operator container automatically mounts the ConfigMap.
```
Comment (rolfedh) [inline modules/ossm-control-plane-profiles.adoc:15]:
```suggestion
To add custom profiles, you must create a `ConfigMap` named `smcp-templates` in the `openshift-operators` project. The Operator container automatically mounts the `ConfigMap`.
```

**Review by rolfedh — APPROVED** [review]

File: modules/ossm-control-plane-profiles.adoc (hunk 1 above)
Comment (rh-tokeefe) [reply modules/ossm-control-plane-profiles.adoc:15]: Thanks for pointing this out, Rolfe. I've made this edit.

**Pull request merged**